pkg/remote/aws/init.go

```diff
@@ -26,7 +27,9 @@ func Init(alerter *alerter.Alerter, providerLibrary *terraform.ProviderLibrary,
 		return err
 	}
 
-	s3Repository := repository.NewS3Repository(client.NewAWSClientFactory(provider.session))
+	repositoryCache := cache.New(100)
```

Comment:
can't we have a bigger cache than 100 ?

Reply:
From what I calculated (~5 methods per repository for 11 repositories), we should not reach more than 55 keys. I set capacity of 100 because we have a lot of margin. I think we can easily increase capacity up to 1000 without any negative impact on performances.

Reply:
Ho you're right I thought at first read this would be number of resource but this will be one by resource type :+1: